### PR TITLE
Look sir, synths with access to Binary radio!

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
This pull request gives synth access to binary using the :n key. Basically the same thing as cyborgs.
The AI actually has it already, but like, no one can actually listen to it lmao.
This also features a comment a check and general prettying up of this check. 

## Why It's Good For The Game
It gives synths and silicon a bit of their own mini group, distinct from command. Kind of like their own channel. 

## Changelog
:cl:
qol: Brought Binary radio for synths
/:cl:

### CODE REVIEW

**Comment by KornFlaks** [comment]
Also give ghosts access to robotic channel please. Just to make sure no ERPing is going on in that channel...

**Comment by Losenis** [comment]
.n cap is condom bolt him in showers

**Comment by lbnesquik** [comment]
> 
> 
> Also give ghosts access to robotic channel please. Just to make sure no ERPing is going on in that channel...

Maybe they should get some synthetic upgrade if they wanted to see it.

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -839,6 +839,10 @@
 	var/datum/atom_hud/AH = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED_SYNTH]
 	AH.add_hud_to(H)
 
+/mob/living/carbon/human/species/synthetic/binarycheck(mob/H)
+	. = ..()
+	return TRUE
```
Comment (SplinterGP) [inline code/modules/mob/living/carbon/human/species.dm:844]:
```suggestion
/mob/living/carbon/human/species/synthetic/binarycheck(mob/H)
	return TRUE
```
no need for this

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -898,6 +902,10 @@
 	AH.remove_hud_from(H)
 	return ..()
 
+/mob/living/carbon/human/species/early_synthetic/binarycheck(mob/H)
+	. = ..()
+	return TRUE
```
Comment (SplinterGP) [inline code/modules/mob/living/carbon/human/species.dm:907]: ditto
```suggestion
/mob/living/carbon/human/species/early_synthetic/binarycheck(mob/H)
	return TRUE
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Ghosts need to be able to see robo chat for admin purposes

**Review by SplinterGP — APPROVED** [review]

**Comment by SplinterGP** [comment]
> 
> 
> Ghosts need to be able to see robo chat for admin purposes

already does ?

File: code/modules/mob/living/silicon/say.dm [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
-	if(issilicon(src))
-		var/mob/living/silicon/S = src
+	if(issilicon(src) || issynth(src))
+		var/mob/living/S = src
 		desig = trim_left(S.job.title)
 	var/message_a = say_quote(message)
 	var/rendered = "Robotic Talk, [span_name("[name]")] [span_message("[message_a]")]"
 	for(var/mob/M in GLOB.player_list)
-		if(M.binarycheck())
+		if(binarycheck(M))
```
Comment (SplinterGP) [inline code/modules/mob/living/silicon/say.dm:17]: ???

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by BraveMole — APPROVED** [review]

**Pull request merged**